common/spec/dependabot/pull_request_creator/message_builder_spec.rb

```diff
@@ -2422,6 +2422,142 @@ def commits_details(base:, head:)
           end
         end
 
+        context "with five or more dependencies with same name, differet to and from versions", :vcr do
```

Comment:
this line has a type for different
do we need the vcr? I don't see one added here

Reply:
Removed all the unnecessary vcr from all the tests. I had copied it from other test which already had the vcr along with manual stub. Good catch.

Reply:
Typo was fixed already: https://github.com/dependabot/dependabot-core/pull/9436/commits/4d241b2d02d95988a25f8e1d248bb1080edc3d92